### PR TITLE
Feat/quorum iot

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

**Comment by akegaviar** [comment]
@ThomasRalee please take one final look

File: docs/tutorials/food-temperature-control-on-quorum.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+* [Chainstack](https://console.chainstack.com/) account to deploy a Quorum network.
+
+## Overview
+
+To get from zero to a deployed Quorum network with a public contract and a private contract, do the following:
+
+* Prepare:
+
+  1. Deploy a Quorum network with Chainstack.
+  1. Install Ethereum JavaScript API to interact with the Quorum network.
```
Comment (ThomasRalee) [inline docs/tutorials/food-temperature-control-on-quorum.md]: I suggest adding the name of the library here?

File: docs/tutorials/food-temperature-control-on-quorum.md [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+[Node2] temp retrieved after updating contract from internal nodes: 12
+[Node3] temp retrieved from external nodes after update null
+```
+
+::: tip See also:
+
+* [Interacting with the blockchain](/guides/interacting-with-the-blockchain#multichain)
+* [Application development](/guides/application-development)
+* [Quorum: Constellation](https://docs.goquorum.com/en/latest/Privacy/Constellation/Constellation/)
+
```
Comment (ThomasRalee) [inline docs/tutorials/food-temperature-control-on-quorum.md]: Looks good, maybe you use some diagrams that i've created
https://hackernoon.com/executing-public-private-transactions-on-jpmorgans-quorum-with-web3-347190e3c35f

File: docs/tutorials/food-temperature-control-on-quorum.md (hunk 1 above)
Comment (akegaviar) [reply docs/tutorials/food-temperature-control-on-quorum.md]: This is just an overview what the user should expect. It goes into detail later in the docs.

File: docs/tutorials/food-temperature-control-on-quorum.md (hunk 2 above)
Comment (akegaviar) [reply docs/tutorials/food-temperature-control-on-quorum.md]: Our approach to diagrams is a topic for separate discussion. They need to be standardized across all docs.

**Pull request merged**